### PR TITLE
Add example table for grace period

### PR DESCRIPTION
**Pull request by fyockm**
image

### CODE REVIEW

File: docs/subscription/index.md [hunk 1]
```diff
@@ -60,15 +60,35 @@ billed $200 USD per month (after the trial period).
 ### Does Buf offer overage protection?
 
 To help protect our customers from unexpected overages, Buf offers the option for billing admins to receive an email
-alert whenever the max usage for their organization has progressed into the next tier. These email alerts are enabled by
-default and will be sent when the next tier is surpassed. Although we will **not** block uploading of more Types, admins
-will have the ability to discuss with their organization and adjust if necessary.
+alert whenever the usage for their organization has progressed into the next level. These email alerts are enabled by
+default and will be sent when usage has passed the next increment of 100 Types. Although we will **not** block uploading
```
Comment (maryelizbeth) [inline docs/subscription/index.md:64]:
```suggestion
default and are sent when usage exceeds the next increment of 100 Types. Although we will **not** block uploading
```

File: docs/subscription/index.md [hunk 2]
```diff
@@ -60,15 +60,35 @@ billed $200 USD per month (after the trial period).
...
+of more Types, admins will have the ability to discuss with their organization and adjust if necessary.
 
 If admins wish to receive alert emails but prefer to decrease noise, a higher threshold can be set before messages are
 sent. Assume a customer is on the Pro plan and has a desired budget of $500 USD. The alert threshold can be set at 500
 Types, and no alerts will be sent until they reach that level of usage. An email alert will be sent at each level beyond
 the chosen threshold.
 
+Additionally, a grace period of 1 billing cycle is automatically applied before customers are invoiced for a higher
```
Comment (maryelizbeth) [inline docs/subscription/index.md:72]:
```suggestion
Additionally, a grace period of 1 billing cycle is automatically applied before you are invoiced for a higher
```

File: docs/subscription/index.md [hunk 3]
```diff
@@ -60,15 +60,35 @@ billed $200 USD per month (after the trial period).
...
+amount. In practice, the grace period means you will be invoiced for the lower number of max types between current and
+previous billing cycles. To illustrate the grace period, see the table below.
+
+### Usage Example
+
+Assume a customer subscribes to the Pro plan in March.
```
Comment (maryelizbeth) [inline docs/subscription/index.md:78]:
```suggestion
Assume you subscribe to the Pro plan in March.
```

**Review by maryelizbeth — COMMENTED** [review]
small comments, feel free to disregard!

**Review by maryelizbeth — APPROVED** [review]

**Comment by fyockm** [comment]
moved to https://github.com/bufbuild/buf.build/pull/553

**Pull request closed without merging**